contracts/src/oracle/IOracle.cairo

```diff
@@ -129,4 +129,9 @@ namespace IOracle {
 
     func set_sources_threshold(threshold: felt) {
     }
+
+    func get_last_checkpoint_before(key: felt, timestamp: felt) -> (
```

Comment:
Feels confusing to call this `timestamp` but to call it `start_tick` in the underlying library - wdyt?

Reply:
Not sure if it matters, but curious what you think. Feel quite strongly it should be called `timestamp` on the outward facing call given my intuition and that the Carmine guy asked what `start_tick` meant.

Reply:
Since this is expected to be the x-axis, start tick to me meant the start point in your time series.  I'll rename it though, since this is just looking up a single point and has no concept of a series.